com.brave.Brave.desktop

```diff
@@ -0,0 +1,221 @@
+[Desktop Entry]
```

Comment:
should be contributed upstream.

Reply:
it is from upstream but I had to change the `icon=` part to `com.brave.Brave`

Reply:
Use rename-icon instead of copying the file here then.

Reply:
is it possible for me to change, `Exec=` too? I checked the original and it seems like they used an absolute path to `/usr/bin/brave-browser`

Reply:
You can use `desktop-file-edit` for that.